src/Checkbox/Checkbox.js

```diff
@@ -31,7 +31,7 @@ function Checkbox({
       checked={!!child}
       disabled={disabled}
       size={size}
-      kind={disabled ? 'grayBase' : kind}
```

Comment:
move all color-related logics in `styled-components.js` to make it tidier
(color, bg-color, border-color)